domain/domain.go

```diff
@@ -354,10 +354,10 @@ func (do *Domain) Reload() error {
 	}
 
 	var (
-		fullLoad        bool
-		changedTableIDs []int64
+		fullLoad                bool
+		changedPhysicalTableIDs []int64
 	)
-	neededSchemaVersion, changedTableIDs, fullLoad, err = do.loadInfoSchema(do.infoHandle, schemaVersion, ver.Ver)
```

Comment:
We change this variable name, then do we need to change the argument name of `changedTableIDs` in`schemaValidator.Update`?